runtime_lib/airhost/host.cpp

```diff
@@ -462,7 +462,11 @@ hsa_status_t air_iterate_agents(hsa_status_t (*callback)(air_agent_t agent,
       return HSA_STATUS_ERROR;
 
     uint64_t *bram_base = reinterpret_cast<uint64_t *>(
-        mmap(NULL, 0x1000, PROT_READ | PROT_WRITE, MAP_SHARED, fd, 0));
+        mmap(NULL, 256*1024, PROT_READ | PROT_WRITE, MAP_SHARED, fd, 0));
+
+    // Platform version is a constant register after the BRAM
+    uint32_t platform_version = (uint32_t)bram_base[PLATFORM_VERSION_REG / sizeof(uint64_t)];
+    std::cout << "Platform version: " << platform_version << std::endl;
```

Comment:
In general the runtime shouldn't print anything. Can we put this behind a tested C API for now? and maybe an hsa_agent_info query later?

Reply:
Good point and inline with Phil's comment. Will do.

Reply:
Printing done in commit b644d1c. Adding to an agent info query and tested in 271 in commit fca3217.